User profile is available. Using '"C:\Users\Nick\AppData\Local\ASP.NET\DataProtection-Keys"' as key repository and Windows DPAPI to encrypt keys at rest.
Now listening on: "https://localhost:5001"
Now listening on: "http://localhost:5000"
Application started. Press Ctrl+C to shut down.
Hosting environment: "Development"
Content root path: "D:\Checks\Logging\Logging"
Request starting HTTP/2 GET https://localhost:5001/  
Executing endpoint '"Logging.HomeController.Index (Logging)"'
Route matched with "{action = \"Index\", controller = \"Home\", page = \"\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult Index()" on controller "Logging.HomeController" ("Logging").
Exception thrown when trying to convert customer viewmodel to model or getting data from the database with id: 1231312
An error accured: System.Exception: LoOL
 ---> System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
   --- End of inner exception stack trace ---
   at Logging.ExceptionService.ThrowException() in D:\Checks\Logging\Logging\ExceptionService.cs:line 9
   at Logging.HomeController.Index() in D:\Checks\Logging\Logging\HomeController.cs:line 24.
An error accured
System.Exception: LoOL
 ---> System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
   --- End of inner exception stack trace ---
   at Logging.ExceptionService.ThrowException() in D:\Checks\Logging\Logging\ExceptionService.cs:line 9
   at Logging.HomeController.Index() in D:\Checks\Logging\Logging\HomeController.cs:line 24
Executing ViewResult, running view "Index".
Executed ViewResult - view "Index" executed in 24.8418ms.
Executed action "Logging.HomeController.Index (Logging)" in 48.7311ms
Executed endpoint '"Logging.HomeController.Index (Logging)"'
Request finished in 100.1867ms 200 text/html; charset=utf-8
Application is shutting down...
User profile is available. Using '"C:\Users\Nick\AppData\Local\ASP.NET\DataProtection-Keys"' as key repository and Windows DPAPI to encrypt keys at rest.
Now listening on: "https://localhost:5001"
Now listening on: "http://localhost:5000"
Application started. Press Ctrl+C to shut down.
Hosting environment: "Development"
Content root path: "D:\Checks\Logging\Logging"
Request starting HTTP/2 GET https://localhost:5001/  
Executing endpoint '"Logging.HomeController.Index (Logging)"'
Route matched with "{action = \"Index\", controller = \"Home\", page = \"\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult Index()" on controller "Logging.HomeController" ("Logging").
Exception thrown when trying to convert customer viewmodel to model or getting data from the database with id: 1231312
An error accured: System.Exception: LoOL
 ---> System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
   --- End of inner exception stack trace ---
   at Logging.ExceptionService.ThrowException() in D:\Checks\Logging\Logging\ExceptionService.cs:line 9
   at Logging.HomeController.Index() in D:\Checks\Logging\Logging\HomeController.cs:line 24.
An error accured
System.Exception: LoOL
 ---> System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
   --- End of inner exception stack trace ---
   at Logging.ExceptionService.ThrowException() in D:\Checks\Logging\Logging\ExceptionService.cs:line 9
   at Logging.HomeController.Index() in D:\Checks\Logging\Logging\HomeController.cs:line 24
Executing ViewResult, running view "Index".
Executed ViewResult - view "Index" executed in 24.7667ms.
Executed action "Logging.HomeController.Index (Logging)" in 45.9712ms
Executed endpoint '"Logging.HomeController.Index (Logging)"'
Request finished in 92.4194ms 200 text/html; charset=utf-8
Application is shutting down...
